vcd/resource_vcd_data_center_group.go

```diff
@@ -0,0 +1,452 @@
+)
+
+var participatingOrgVdcsResource = &schema.Resource{
+	Schema: map[string]*schema.Schema{
+		"fault_domain_tag": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Represents the fault domain of a given organization vDC.",
+		},
+		"network_provider_scope": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Specifies the network provider scope of the vDC.",
+		},
+		"remote_org": {
+			Type:        schema.TypeBool,
+			Computed:    true,
+			Description: "Specifies whether the vDC is local to this VCD site",
+		},
+		"status": {
+			Type:     schema.TypeString,
+			Computed: true,
+			Description: "The status that the vDC can be in e.g. 'SAVING', 'SAVED', 'CONFIGURING'," +
+				" 'REALIZED', 'REALIZATION_FAILED', 'DELETING', 'DELETE_FAILED', 'OBJECT_NOT_FOUND'," +
+				" 'UNCONFIGURED').",
+		},
+		"org_name": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Organization VDC belongs",
+		},
+		"org_id": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Organization VDC belongs",
+		},
+		"site_name": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Site VDC belongs",
+		},
+		"site_id": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "Site VDC belongs",
+		},
+		"vdc_name": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "VDC name",
+		},
+		"vdc_id": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "VDC ID",
+		},
+	},
+}
+
+func resourceDataCenterGroup() *schema.Resource {
+	return &schema.Resource{
+		ReadContext:   resourceVcdDataCenterGroupRead,
+		CreateContext: resourceVcdDataCenterGroupCreate,
+		UpdateContext: resourceVcdDataCenterGroupUpdate,
+		DeleteContext: resourceVcdAlbDataCenterGroupDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceDataCenterGroupImport,
+		},
+		Schema: map[string]*schema.Schema{
+			"org": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Description: "The name of organization to use, optional if defined at provider " +
+					"level. Useful when connected as sysadmin working across different organizations",
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Name of data center group",
+			},
+			"description": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "Data center group description",
+			},
+			"dfw_enabled": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Computed:    true,
+				Description: "Distributed firewall status",
+			},
+			"starting_vdc_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "Starting VDC Id",
+			},
+			"participating_vdc_ids": {
+				Type:        schema.TypeSet,
+				Required:    true,
```

Comment:
I put the comment in docs, but here as well - can a VDC group only contain a `starting_vdc_id`?

Reply:
answer is the same